grants_tagger/filter_disease_codes.py

```diff
@@ -29,6 +30,20 @@ def filter_disease_codes(mesh_descriptions_file, mesh_export_file):
     argparser = ArgumentParser(description=__file__)
     argparser.add_argument('--mesh_descriptions_file', type=Path, help="path to xml file containing MeSH taxonomy")
     argparser.add_argument('--mesh_export_file', type=Path, help="path to csv file to export Mesh terms")
+    argparser.add_argument('--config', type=Path, help="path to config file defining arguments")
     args = argparser.parse_args()
 
-    filter_disease_codes(args.mesh_descriptions_file, args.mesh_export_file)
+    if args.config:
+        config_parser = ConfigParser()
+        cfg = config_parser.read(args.config)
+
+        mesh_descriptions_file = cfg["filter_disease_codes"]["mesh_descriptions_file"]
+        mesh_export_file = cfg["filter_disease_codes"][""]
```

Comment:
Is this correct? (the last bracket `[""]`)

Reply:
Hmm, that seems weird, let me investigate

Reply:
Good find, not on purpose.

Resolved by 45ebc17c0cc4fb413c283dd0f6a664bf4f0d2368